src/main/java/redgatesqlci/Utils.java

```diff
@@ -5,34 +5,71 @@
 import hudson.Proc;
 import hudson.model.AbstractBuild;
 import hudson.model.BuildListener;
+import hudson.remoting.Callable;
+import hudson.remoting.VirtualChannel;
 
 import java.io.*;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.HashMap;
 import java.util.Map;
 
+import org.jenkinsci.remoting.RoleChecker;
+
 public class Utils {
+    private static String GetEnvironmentVariable(final String variableName, VirtualChannel channel)
```

Comment:
We should follow Java lower camelcase rules for the names of these private methods, too.